tools/eslint-plugin-cdk/test/rules/fixtures.test.ts

```diff
@@ -67,3 +87,19 @@ async function lintAndFix(file: string, outputDir: string) {
   }
   return newPath;
 }
+
+async function lintAndGetErrorCountAndMessage(file: string) {
+  const result = await linter.lintFiles(file);
+  let errorCount = 0;
+  let errorMessage: string | undefined = undefined;
+  if (result.length === 1) {
+    errorCount = result[0].errorCount;
+    if (result[0].messages.length === 1) {
+      errorMessage = result[0].messages[0].message;
+    }
+  };
+  return {
+    errorCount,
+    errorMessage,
+  };
```

Comment:
I'm a bit confused here. Why not just return all error messages as an array?

Reply:
This was because I restricted it to just one error message per test. I can extend this to support multiple error messages in an `error.txt` file, like you suggested in the other comment.

Reply:
Fixed